.github/services/hdfs/hdfs_default_azblob/action.yml

```diff
@@ -0,0 +1,67 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+name: hdfs_default_azblob
```

Comment:
s3 has `minio` setup and `azblob` has `azurite` setup. They can be tested even in forked repo. Would you like to use the same setup?

Reply:
Do you agree we should rename the test cases and file names accordingly? E.g.
- `hdfs_default_s3` -> `hdfs_default_minio_s3`
- `s3-core-site.xml` -> `minio-s3-core-site.xml`

would be more specific

Reply:
> Do you agree we should rename the test cases and file names accordingly?

Good idea, we can use `hdfs_default_on_minio_s3` and  `minio-s3-core-site.xml`.